src/App.js

```diff
@@ -1,5 +1,35 @@
+import "./App.css";
+import { Route, Routes } from "react-router-dom";
+import Header from "./frontend/components/Navbar/Navbar";
+import ResponsiveNavbar from "./frontend/components/ResponsiveNavbar/ResponsiveNavbar";
+import HomeScreen from "./frontend/screens/HomeScreen/HomeScreen";
+import AboutUsScreen from "./frontend/screens/AboutUsScreen/AboutUsScreen";
+import ProductsScreen from "./frontend/screens/ProductsScreen/ProductsScreen";
+import CartScreen from "./frontend/screens/CartScreen/CartScreen";
+import WishlistScreen from "./frontend/screens/WishlistScreen/WishlistScreen";
+import SignInScreen from "./frontend/screens/SignInScreen/SignInScreen";
+import SignUpScreen from "./frontend/screens/SignUpScreen/SignUpScreen";
```

Comment:
we can use barrel export here. create an index.js in the screens folder & export all the screens from index.js.

Reply:
Absolutely right! Doing it now. Thank you.